cub/cub/device/dispatch/tuning/tuning_unique_by_key.cuh

```diff
@@ -344,364 +320,273 @@ template <class KeyT,
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::fixed_delay_constructor_t<964, 1125>;
 };
 
 template <class KeyT, class ValueT>
-struct sm80_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::yes, key_size::_8, val_size::_4>
+struct sm90_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::yes, key_size::_8, val_size::_4>
 {
-  static constexpr int threads = 256;
-
-  static constexpr int items = 7;
-
+  static constexpr int threads                       = 640;
+  static constexpr int items                         = 7;
   static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_DIRECT;
-
-  static constexpr CacheLoadModifier load_modifier = LOAD_DEFAULT;
-
-  using delay_constructor = detail::fixed_delay_constructor_t<740, 1105>;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::no_delay_constructor_t<1070>;
 };
 
 template <class KeyT, class ValueT>
-struct sm80_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::yes, key_size::_8, val_size::_8>
+struct sm90_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::yes, key_size::_8, val_size::_8>
 {
-  static constexpr int threads = 192;
-
-  static constexpr int items = 7;
-
-  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_DIRECT;
-
-  static constexpr CacheLoadModifier load_modifier = LOAD_DEFAULT;
-
-  using delay_constructor = detail::fixed_delay_constructor_t<764, 1155>;
+  static constexpr int threads                       = 448;
+  static constexpr int items                         = 11;
+  static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_WARP_TRANSPOSE;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::no_delay_constructor_t<1190>;
 };
 
 template <class KeyT, class ValueT>
-struct sm80_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::no, key_size::_8, val_size::_16>
+struct sm90_tuning<KeyT, ValueT, primitive_key::yes, primitive_val::no, key_size::_8, val_size::_16>
 {
-  static constexpr int threads = 128;
-
-  static constexpr int items = 7;
-
+  static constexpr int threads                       = 256;
+  static constexpr int items                         = 9;
   static constexpr BlockLoadAlgorithm load_algorithm = BLOCK_LOAD_WARP_TRANSPOSE;
-
-  static constexpr CacheLoadModifier load_modifier = LOAD_DEFAULT;
-
-  using delay_constructor = detail::fixed_delay_constructor_t<992, 1135>;
+  static constexpr CacheLoadModifier load_modifier   = LOAD_DEFAULT;
+  using delay_constructor                            = detail::no_delay_constructor_t<1155>;
 };
 
-} // namespace unique_by_key
-} // namespace detail
-
-template <typename KeyInputIteratorT, typename ValueInputIteratorT = unsigned long long int*>
-struct DeviceUniqueByKeyPolicy
```

Comment:
Important: we cannot drop `cub::DeviceUniqueByKeyPolicy` because it's part of the public API. We need to keep an alias below. See later comment.